azafea/event_processors/endless/metrics/events/__init__.py

```diff
@@ -76,6 +76,26 @@ class CacheMetadataIsCorrupt(SingularEvent):
     __payload_type__ = None
 
 
+class ControlCenterAutomaticUpdates(SingularEvent):
+    __tablename__ = 'control_center_automatic_updates'
+    __event_uuid__ = '510f9741-823e-41a9-af2d-048895f990c0'
+    __payload_type__ = '(bbbv)'
+
+    allow_downloads_when_metered = Column(Boolean, nullable=False)
+    automatic_updates_enabled = Column(Boolean, nullable=False)
+    tariff_enabled = Column(Boolean, nullable=False)
+    tariff_variant = Column(Unicode, nullable=False)
```

Comment:
Not json(b)?

Reply:
(I believe the type is something like `(sua(suu))` with no types that should be problematic to convert to JSON?)

Reply:
That’s a better solution, thanks. Even easier once I found [`GLib.Variant.unpack()`](https://lazka.github.io/pgi-docs/GLib-2.0/classes/Variant.html#GLib.Variant.unpack) to convert `GLib.Variant` to native Python types.

Reply:
Pygobject will also implicitly unpack it if you do various pythonic operations in it. Better to explicit though.